api/src/pcapi/routes/serialization/statistics_serialize.py

```diff
@@ -9,13 +11,17 @@ class StatisticsQueryModel(BaseModel):
 class StatisticsModel(BaseModel, YearlyAggregatedRevenueModel):
     @classmethod
     def from_query(cls, income_by_year: dict) -> "StatisticsModel":
+        current_year = datetime.utcnow().year
         if income_by_year.keys():
             min_year = int(min(income_by_year.keys()))
             max_year = int(max(income_by_year.keys()))
 
             for year in range(min_year, max_year + 1):
                 if str(year) not in income_by_year:
                     income_by_year[str(year)] = {}
+                # we don't need to serialize expected_revenue in previous years
+                elif year != current_year and hasattr(income_by_year[str(year)], "expected_revenue"):
```

Comment:
Pour pinailler, tu parles d'« années passées », donc `<`, sinon ce serait « autres années » :slightly_smiling_face: 
```suggestion
                elif year < current_year and hasattr(income_by_year[str(year)], "expected_revenue"):
```